src/transformers/models/canine/modeling_canine.py

```diff
@@ -52,6 +52,16 @@
 _CONFIG_FOR_DOC = "CanineConfig"
 _TOKENIZER_FOR_DOC = "CanineTokenizer"
 
+# Base model docstring
+_EXPECTED_OUTPUT_SHAPE = [1, 8, 768]
```

Comment:
Hey Niels, sorry we changed the template just today since just testing for shapes doesn't really make sense IMO. Could you adapt the expected values according to https://github.com/huggingface/transformers/pull/16363/files#r833804669

Wrote a guide here: https://github.com/huggingface/transformers/issues/16292#issuecomment-1076919075 on how to find good checkpoins as well. If Canine doesn't have any good fine-tuned checkpoints for a task, I'd recommend to just upload one with a random head and use this to make the test not fail due to random values

Reply:
To avoid confusion, @patrickvonplaten  is talking about the `...ForXXX` models for downstream tasks, not the base model.